src/components/Room.tsx

```diff
@@ -24,8 +27,10 @@ function Room() {
 
     function onTimerUpdate(value: string) {
       console.log("timer", value);
-      setTimestamp(value);
+      const timer = parseInt(value);
+      setTimestamp(timer);
```

Comment:
This can probably be a single line
```suggestion
      setTimestamp(parseInt(value));
```